src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationAndGeneratorDriverTranslationAction_Actions.cs

```diff
@@ -200,14 +186,6 @@ public AddAdditionalDocumentsAction(ImmutableArray<TextDocumentState> additional
                 {
                     _additionalDocuments = additionalDocuments;
```

Comment:
There is `GeneratorDriver.{Add,Remove}AdditionalTexts()` for this scenario.

Reply:
Ditto: we're not really doing incremental use of this at this point, so it doesn't seem helpful? I agree we'll go back to that at some point...